<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="navigation" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QPushButton" name="b_user">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_calendar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Calendar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_events">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Events</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_settings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_sidebar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sidebar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="container" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>660</width>
      <height>460</height>
     </rect>
    </property>
    <widget class="QStackedWidget" name="pages">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>660</width>
       <height>460</height>
      </rect>
     </property>
     <widget class="QWidget" name="page_selectUser">
      <widget class="QStackedWidget" name="userinfo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>315</width>
         <height>420</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="p_user_login">
        <widget class="QLineEdit" name="tb_Username">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>300</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Enter your username here</string>
         </property>
        </widget>
        <widget class="QLabel" name="l_Login">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>291</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="tb_Password">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>274</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::InputMethodHint::ImhHiddenText</set>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Enter your password here</string>
         </property>
        </widget>
        <widget class="QPushButton" name="b_showhidebutton">
         <property name="geometry">
          <rect>
           <x>286</x>
           <y>200</y>
           <width>24</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="l_Username">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QLabel" name="l_Password">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>300</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
        <widget class="QPushButton" name="b_confirm">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>250</y>
           <width>100</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="p_user_profile">
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>330</y>
           <width>100</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>120</y>
           <width>58</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>User</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>160</y>
           <width>58</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="Profile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>330</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="Select">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>330</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="QPushButton" name="timebutton">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>230</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Open TimeDia</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_calendarTab">
      <widget class="QScrollArea" name="upcomingevents">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>340</width>
         <height>440</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="EventListView" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>322</width>
          <height>438</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="b_delete">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>120</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
      <widget class="QComboBox" name="userSelect">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>181</width>
         <height>32</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Select User</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="pages_addEvent">
      <widget class="QScrollArea" name="eventform">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>0</y>
         <width>341</width>
         <height>461</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>339</width>
          <height>459</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>331</width>
           <height>407</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="generalcontainer">
          <item>
           <layout class="QFormLayout" name="GenericSettings">
            <item row="0" column="0">
             <widget class="QLabel" name="AE_label_userSelectCB">
              <property name="text">
               <string>User</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="AE_CB_userSelect"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="AE_label_titleLE">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="AE_LE_inputTitle">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="AE_label_startDT">
              <property name="text">
               <string>StartDate</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateTimeEdit" name="AE_DTE_startDTSelect">
              <property name="currentSection">
               <enum>QDateTimeEdit::Section::DaySection</enum>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="AE_label_endDT">
              <property name="text">
               <string>EndDate</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateTimeEdit" name="AE_DTE_endDTSelect">
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="AE_PB_expandUsers">
            <property name="text">
             <string>Add Users (Expand)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="AE_LV_addUsers"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="AE_container_addJSON">
            <item>
             <widget class="QPushButton" name="expandcomponent">
              <property name="text">
               <string>Add Components</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="select_add">
              <item>
               <widget class="QComboBox" name="selectcomponenttype"/>
              </item>
              <item>
               <widget class="QPushButton" name="addcomponent">
                <property name="text">
                 <string>add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="dynamiccomponentsection"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="AE_PB_addEvent">
            <property name="text">
             <string>AddEvent</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_settingsTab">
      <widget class="QTableView" name="table_user">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="table_user_profile">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="table_tags">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="table_events">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>240</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventListView</class>
   <extends>QWidget</extends>
   <header>views.resources.custom_classes.eventlistview</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
